### DISCUSSION TITLE
Maximum size limit

### DISCUSSION THREAD

**Discussion by arinteckk** [vercel/community#2357] [Help] [answered]
When i try to deploy laravel project i get : 
`Error: The Serverless Function "public/index.php" is 113.71mb which exceeds the maximum size limit of 50mb.`

**Comment by amyegan** [comment, marked as answer]
Hi @arinteckk! It looks like you're using the [community-developed PHP Runtime](https://vercel.com/docs/concepts/functions/serverless-functions/runtimes#community-runtimes) and exceeding the 50MB [Serverless Function Size](https://vercel.com/docs/concepts/limits/overview#serverless-function-size) limit. We have a large discussion where people are sharing various solutions and workarounds that may help you: https://github.com/vercel/community/discussions/103

**Reply by arinteckk** [reply]
OK thank you !
